features/frameworks/greentea-client/greentea-client/test_env.h

```diff
@@ -28,6 +28,9 @@
 #endif
 
 #include <stdio.h>
+extern "C" {
```

Comment:
I would prefer a standard runtime of getc/putc. But I don't intend to change existing implementation here. I need Greentea API for the mbedtl C test code. 
Are you suggesting C implementation of Greentea API. It seems reasonable since it has no C++.